### PR TITLE
Takeabreak software

### PR DESCRIPTION
**Pull request by cmonjeau**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/takeabreak/build.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+mkdir -p ${PREFIX}/bin
+
+# decrease RAM needed
+sed -i.bak 's/make -j4/make -j1/' INSTALL
+sed -i.bak 's/.\/build\/bin\/TakeABreak/.\/build\/bin\/TakeABreak -max-memory 1000/' INSTALL
```
Comment (bgruening) [inline recipes/takeabreak/build.sh]: why is this may-memory needed?

Comment (bgruening) [reply recipes/takeabreak/build.sh]: Is travis not handling this well?

Comment (bgruening) [reply recipes/takeabreak/build.sh]: I hope so. They are limited, but we never had problems before.

Comment (cmonjeau) [reply recipes/takeabreak/build.sh]: Ohoho. I have to increase my virtual machine for my developments :).

**Comment by cmonjeau** [comment]
👍

**Pull request merged**